lang/performous.pot

```diff
@@ -490,863 +575,3 @@ msgstr ""
-msgid "Some devices failed to open!"
-msgstr ""
-
-#: ../game/screen_intro.cc:25
-msgid "No playback devices could be used.\n"
-msgstr ""
-
-#: ../game/screen_intro.cc:26
-msgid ""
-"\n"
-"Please configure some before playing."
-msgstr ""
-
-#: ../game/screen_intro.cc:74 ../game/screen_paths.cc:36
-msgid "Settings saved as system defaults."
-msgstr ""
-
-#: ../game/screen_intro.cc:74 ../game/screen_paths.cc:36
-msgid "Settings saved."
-msgstr ""
-
-#: ../game/screen_intro.cc:154
-msgid "Ctrl + S to save, Ctrl + R to reset defaults"
-msgstr ""
-
-#: ../game/screen_intro.cc:172
-msgid "Perform"
-msgstr ""
-
-#: ../game/screen_intro.cc:173
-msgid "Practice"
-msgstr ""
-
-#: ../game/screen_intro.cc:173
-msgid "Check your skills or test the microphones"
-msgstr ""
-
-#: ../game/screen_intro.cc:189
-msgid "Configure"
-msgstr ""
-
-#: ../game/screen_intro.cc:189
-msgid "Configure audio and game options"
-msgstr ""
-
-#: ../game/screen_intro.cc:190
-msgid "Leave the game"
-msgstr ""
-
-#: ../game/screen_intro.cc:200
-msgid ""
-"Webserver active!\n"
-" use a webbrowser\n"
-"and navigate to localhost:"
-msgstr ""
-
-#: ../game/screen_intro.cc:207
-msgid ""
-"Webserver active!\n"
-" connect to this computer\n"
-"using "
-msgstr ""
-
-#: ../game/screen_paths.cc:69
-msgid "Hide hidden folders"
-msgstr ""
-
-#: ../game/screen_paths.cc:81
-msgid "Remove this folder"
-msgstr ""
-
-#: ../game/screen_paths.cc:81
-msgid "Remove current folder from song folders"
-msgstr ""
-
-#: ../game/screen_paths.cc:87
-msgid "Add this folder"
-msgstr ""
-
-#: ../game/screen_paths.cc:87
-msgid "Add current folder to song folders"
-msgstr ""
-
-#: ../game/screen_paths.cc:93
-msgid ".."
-msgstr ""
-
-#: ../game/screen_paths.cc:93
-msgid "Go up one folder"
-msgstr ""
-
-#: ../game/screen_paths.cc:104
-msgid "Open folder"
-msgstr ""
-
-#: ../game/screen_players.cc:117
-msgid "No players found!"
-msgstr ""
-
-#: ../game/screen_players.cc:118
-msgid "Enter a name to create a new player."
-msgstr ""
-
-#: ../game/screen_players.cc:120
-msgid "Press enter to create player!"
-msgstr ""
-
-#: ../game/screen_players.cc:124
-msgid "No players worth mentioning!"
-msgstr ""
-
-#: ../game/screen_players.cc:129
-msgid "You reached %1% points!"
-msgstr ""
-
-#: ../game/screen_players.cc:130
-msgid "Change player with arrow keys."
-msgstr ""
-
-#: ../game/screen_players.cc:131
-msgid "Name:"
-msgstr ""
-
-#: ../game/screen_players.cc:134
-msgid "Type text to filter or create a new player."
-msgstr ""
-
-#: ../game/screen_players.cc:134
-msgid "Search Text:"
-msgstr ""
-
-#: ../game/screen_playlist.cc:21 ../game/screen_sing.cc:47
-msgid "Initializing webcam..."
-msgstr ""
-
-#: ../game/screen_playlist.cc:46
-msgid "Loading song timestamps..."
-msgstr ""
-
-#: ../game/screen_playlist.cc:171 ../game/screen_playlist.cc:334
-msgid "Continue"
-msgstr ""
-
-#: ../game/screen_playlist.cc:171
-msgid "Continue playing"
-msgstr ""
-
-#: ../game/screen_playlist.cc:179
-msgid "Add songs"
-msgstr ""
```

Comment:
There is still a lot of deletions.. did you compile with all options on?

Reply:
poedit apparently ignores `*.cpp` files, maybe this update? (with help of `xgettext` inside console)